runtime/kusama/src/weights/pallet_staking.rs

```diff
@@ -214,4 +214,10 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
 			.saturating_add(T::DbWeight::get().reads((1 as Weight).saturating_mul(w as Weight)))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
+	fn get_npos_voters(v: u32, n: u32, s: u32, ) -> Weight {
+		0
```

Comment:
why is it 0 ?

Reply:
because I think there's no way now to run the bench bot unless if the PR is merged :D https://github.com/paritytech/polkadot/pull/2622#issuecomment-799691416

this is one of those cases where I think we have to manually merge.

Reply:
ah yes